c++/src/kj/async.h

```diff
@@ -570,21 +570,6 @@ class CaptureByMove {
   MovedParam param;
 };
 
-template <typename Func, typename MovedParam>
-inline CaptureByMove<Func, Decay<MovedParam>> mvCapture(MovedParam&& param, Func&& func) {
```

Comment:
I suspect some dependents may use this, we should keep it but mark it deprecated.

Reply:
Makes sense! I could not use `KJ_DEPRECATED` in the function declaration so I had to add a definition above, not sure if there is a cleaner way...